app-foundation/3-networks-extension/modules/fw-rules/main.tf

```diff
@@ -59,6 +59,12 @@ locals {
       target_tags        = ["boa-mci-cluster"]
       allow_protocol     = "tcp"
       allow_ports        = ["443", "10250"]
+    },
+    "fw-${var.environment_code}-shared-base-e-bastion-allow-all" = {
+      destination_ranges = ["0.0.0.0/0"]
+      target_tags        = ["bastion"]
+      allow_protocol     = "all"
+      allow_ports        = []
```

Comment:
shouldnt this be `null` to allow all ports?

Reply:
[] should also work but changed to be safe